causalpy/tests/test_pymc_models.py

```diff
@@ -123,3 +123,38 @@ def test_idata_property():
     )
     assert hasattr(result, "idata")
     assert isinstance(result.idata, az.InferenceData)
+
+
+def test_result_reproducibility():
+    """Test that we can reproduce the results from the model. We could in theory test
+    this with all the model and experiment types, but what is being targetted is
+    the ModelBuilder.fit method, so we should be safe testing with just one model. Here
+    we use the DifferenceInDifferences experiment class."""
+    # Load the data
+    df = cp.load_data("did")
+    # Set a random seed
+    sample_kwargs["random_seed"] = 42
```

Comment:
Maybe you can parametrize this test, passing maybe the other two seeds?

Reply:
Great idea. Have pushed changes to do this.